### PR TITLE
Various Changes

### PR DESCRIPTION
**Pull request by GaryJBlake**
- Code Clean Up and Formatting
- New Internal Function createHeader
- New cmdlet Set-VCFFederation
- New Docs for some existing Federation cmdlets

### CODE REVIEW

File: powerVCF.psm1 [hunk 1]
```diff
@@ -3228,247 +3008,260 @@ Export-ModuleMember -Function Get-VCFvROPs
...
 #>
-# Get VCF Version
-CheckVCFVersion
 
-$headers = @{"Accept" = "application/json"}
-$headers.Add("Authorization", "Basic $base64AuthInfo")
-$uri = "https://$sddcManager/v1/sddc-federation"
-try {
+  Try {
+    CheckVCFVersion # Calls Funxtion CheckVCFVersion to check VCF Version
```
Comment (PowerVCF) [inline powerVCF.psm1]: Typo Funxtion + 5 more instances

File: powerVCF.psm1 [hunk 2]
```diff
@@ -2620,30 +2454,26 @@ Function Get-VCFManager {
...
-  $headers.Add("Authorization", "Basic $base64AuthInfo")
-  $method = "GET"
-
-  try {
+  Try {
+    createHeader # Calls Function createHeader to set Accept & Authorization
     if ($PsBoundParameters.ContainsKey("id")) {
       $uri = "https://$sddcManager/v1/sddc-managers/$id"
-      $response = Invoke-RestMethod -Method $method -URI $uri -headers $headers
+      $response = Invoke-RestMethod -Method GETT -URI $uri -headers $headers
```
Comment (PowerVCF) [inline powerVCF.psm1]: Typo GETT :)

File: powerVCF.psm1 (hunk 1 above)
Comment (GaryJBlake) [reply powerVCF.psm1]: Fixed

File: powerVCF.psm1 (hunk 2 above)
Comment (GaryJBlake) [reply powerVCF.psm1]: Fixed

**Pull request merged**